packages/yarnpkg-parsers/sources/grammars/shell.pegjs

```diff
@@ -176,7 +176,6 @@ GlobText
 
 EnvVariable
   = [a-zA-Z0-9_]+ { return text() }
-  / [A-Za-z0-9_]+ { return text() }
```

Comment:
This change was present in https://github.com/yarnpkg/berry/commit/c225c1f6cce5690c2b18d43155932bfd308dbe3c but without any tests or explanation so out it goes.

Reply:
Yep it was just a no-op to change the generated syntax, to check how it merged